operators/pkg/instctrl/containers_test.go

```diff
@@ -102,11 +103,21 @@ var _ = Describe("Generation of the container based instances", func() {
 			&clv1alpha2.Template{ObjectMeta: metav1.ObjectMeta{Name: templateName, Namespace: templateNamespace}},
 		)
 
+		tenant = clv1alpha2.Tenant{
+			ObjectMeta: metav1.ObjectMeta{Name: tenantName},
+			Status: clv1alpha2.TenantStatus{
+				PersonalNamespace: clv1alpha2.NameCreated{
+					Created: true,
+					Name:    instanceNamespace,
+				},
+			},
+		}
 		instance = clv1alpha2.Instance{
 			ObjectMeta: metav1.ObjectMeta{Name: instanceName, Namespace: instanceNamespace},
 			Spec: clv1alpha2.InstanceSpec{
 				Running:  true,
 				Template: clv1alpha2.GenericRef{Name: templateName, Namespace: templateNamespace},
+				Tenant:   clv1alpha2.GenericRef{Name: tenantName},
```

Comment:
A test should be added for when a tenant is not set: the instance should still be able to start if the nfs share mount is not required by the template spec. (Exam instances do not use tenants, the tenant operator is not even deployed)

Reply:
In that case the ```environment.Mode`` is set to ```Standard``` right?
In that case I'm sure it doesn't mount the volume, not even searching for it.
But to be more general: how can I check that the instance has no tenant? The ```instance.Spec.Tenant``` can't be ```nil``` so the ```instance.Spec.Tenant.Name``` is empty string?

Reply:
Correct. `Spec.Tenant` cannot be nil since it's a nested structure. In general we can consider the tenant absent when it's an empty string, it's a rather common approach.

Reply:
Ok added check in the controller for empty string Tenant reference in Instance